odbc/src/api/bind_col_tests.rs

```diff
@@ -581,15 +585,15 @@ mod unit {
             // So I input usize::MAX because it casts to -1 in the function.
             assert_eq!(
                 "aaaa",
```

Comment:
This comment should be removed too. It's the same as the one above that was removed.